Makefile.mingw64.mk

```diff
@@ -30,7 +30,10 @@ build_fftw: $(DEPS_PATH)/fftw
 
 build_libio: $(DEPS_PATH)/liblo
 	cd $< ; \
-	./autogen.sh --prefix=$(PREFIX_PATH) --disable-shared --enable-static
+	./configure --prefix=$(PREFIX_PATH) --disable-shared --enable-static
+	#./autogen.sh --prefix=$(PREFIX_PATH) --disable-shared --enable-static
```

Comment:
I'd say get rid of the commented out code if it doesn't function as expected.

Reply:
Okay.